src/routes/helpers/transformers.js

```diff
@@ -0,0 +1,85 @@
+import SchemaHelper from '../../helpers/SchemaHelper'
+import { warning } from '../../utils/logger'
+import { isDateTime } from 'neo4j-driver/lib/v1/temporal-types.js'
+
+// Used scopes dict
+const scopedContexts = {
+  dc: 'http://purl.org/dc/elements/1.1/',
+  prov: 'https://www.w3.org/TR/prov-o/#',
+  skos: 'http://www.w3.org/2004/02/skos/core#',
+  rdf: 'https://www.w3.org/2000/01/rdf-schema'
+}
+
+/**
+ * Transform document to a JSON-LD structured document
+ * @param {string} type
+ * @param {Object} data
+ * @returns {Object} JSON-LD structured document
+ */
+export const transformJsonLD = (type, data) => {
+  const schemaHelper = new SchemaHelper()
+  const schemaTypes = schemaHelper.getTypeDescription(type).split(',')
+  const prefixes = Object.keys(scopedContexts)
+
+  // Base JSON-LD document
+  const jsonLdData = {
+    '@context': [
+      'https://schema.org/',
+      scopedContexts
+    ],
+    '@type': schemaTypes
+  }
+
+  // Iterate all keys in the data document
+  Object.keys(data).forEach(key => {
+    // Get the property for the current key
+    const property = schemaHelper.findPropertyType(type, key)
+
+    // Use the current value by default
+    let elementValue = data[key];
+
+    // A DateTime object should be formatted as an iso8601 date
+    // TODO: there is also a LocalDateTime, but we don't appear to use it
+    if (isDateTime(elementValue)) {
```

Comment:
This is checking the actual value, rather than the type of the object in the graphql schema (like other checks in the method do, using `property`). I don't think that there's a significant difference between the two ways of doing it, I like this because it uses functionality that's already present in the neo4j js driver.
I'm pretty sure that specifying `_Neo4DateTime` in the graphql schema will always generate a DateTime object here (and not a LocalDateTime). Do we also want to add checks for the other temporal types in neo4j? (Date, Time, LocalTime, Duration: https://neo4j.com/docs/cypher-manual/current/syntax/temporal/)

Reply:
I have no reasons to not test the `elementValue` type... I would indeed try to implement a more generic check for all temporal objects from Neo4j. They all implement the `toString` method, so it can be made into a single check for now.

Reply:
I don't see any method in the neo4j API to check if an element is any one of the temporal types (There is no isTemporal or similar listed at https://neo4j.com/docs/api/javascript-driver/current/variable/index.html). I will make a method that wraps all 6 `is` methods and use that here.

Reply:
What is the status current status? We could merge this PR and add the `isTemporal` check in a separate PR.

Reply:
I won't get it done today. feel free to merge if you want and I'll open the PR next week.